Rubberduck.VBEEditor/Utility/FileExistenceChecker.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.IO;
+
+namespace Rubberduck.VBEditor.Utility
+{
+    public interface IFileExistenceChecker
+    {
+        bool FileExists(string filename);
+    }
+
+    public class FileExistenceChecker : IFileExistenceChecker
+    {
+        public bool FileExists(string filename)
+        {
+            return File.Exists(filename);
+        }
+    }
```

Comment:
feels a bit overkill to me, TBQH, but if it makes unit-testing easier, I'm all for it

Reply:
We can later grow this to a more useful abstraction of the file system. Without this, testing the behaviour regarding missing corresponding binaries is simply impossible.

Reply:
Just to be aware - there are already packages that provides IO abstraction (e.g. `System.IO.Abstraction`) -  we should be careful to not end up reinventing wheel and having to pay additional tech debt as result of reinvented wheel being not to the standard specs.